apps/web/components/RegisterMember.tsx

```diff
@@ -109,123 +78,100 @@ export function RegisterMember({
           containerId: communityAddress,
           function: "unregisterMember",
           id: communityAddress,
-          urlChainId: urlChainId,
+          urlChainId,
```

Comment:
```suggestion
          chainId: urlChainId,
```

Reply:
urlChainId is a custom prop but not exists in the default one:
See pubsub.context.tsx:
```
export type ChangeEventPayload = {
  topic: ChangeEventTopic;
  type?: "add" | "update" | "delete";
  function?: string;
  chainId?: ChainId;
  containerId?: string | number;
  id?: string | number;
} & { [key: string]: Native };
```

Reply:
Sorry it fall on you haha the problem was there before haha